### DISCUSSION TITLE
Availability of Alpine based OpenJDK as container image

### DISCUSSION THREAD

**Discussion by MNicks** [microsoft/openjdk#241] [Q&A] [answered]
Is there a plan to provided the newly announced OpenJDK binaries for Alpine Linux as well as container image?

**Comment by karianna** [comment]
Not at this stage.  Please see https://docs.microsoft.com/en-us/java/openjdk/containers for our current support of containers.

**Comment by karianna** [comment, marked as answer]
https://learn.microsoft.com/en-us/java/openjdk/containers#alpine-images for the guidance on how to create these yourself.
